target/a52q/patches/ascendia/module.prop

```diff
@@ -0,0 +1,4 @@
+id=ascendia
+name=Ascendia Kernel support
+author=BlackMesa123 & RisenID
```

Comment:
Only you in the credits would be better imo

Reply:
Nah, im keeping u in there